diff --git a/dist/main/interfaces/node-api.js b/dist/main/interfaces/node-api.js
index 120c738..0eb035a 100644
--- a/dist/main/interfaces/node-api.js
+++ b/dist/main/interfaces/node-api.js
@@ -5,7 +5,7 @@ const create_root_1 = require("../data/create-root");
 class NodeApi {
     constructor(params) {
         this.handleClick = (e) => {
-            if (e.metaKey && !this.tree.props.disableMultiSelection) {
+            if ((e.metaKey || e.ctrlKey) && !this.tree.props.disableMultiSelection) {
                 this.isSelected ? this.deselect() : this.selectMulti();
             }
             else if (e.shiftKey && !this.tree.props.disableMultiSelection) {
diff --git a/dist/module/components/default-container.js b/dist/module/components/default-container.js
index 47724f5..2285e35 100644
--- a/dist/module/components/default-container.js
+++ b/dist/module/components/default-container.js
@@ -34,28 +34,6 @@ export function DefaultContainer() {
                 return;
             }
             if (e.key === "Backspace") {
-                if (!tree.props.onDelete)
-                    return;
-                const ids = Array.from(tree.selectedIds);
-                if (ids.length > 1) {
-                    let nextFocus = tree.mostRecentNode;
-                    while (nextFocus && nextFocus.isSelected) {
-                        nextFocus = nextFocus.nextSibling;
-                    }
-                    if (!nextFocus)
-                        nextFocus = tree.lastNode;
-                    tree.focus(nextFocus, { scroll: false });
-                    tree.delete(Array.from(ids));
-                }
-                else {
-                    const node = tree.focusedNode;
-                    if (node) {
-                        const sib = node.nextSibling;
-                        const parent = node.parent;
-                        tree.focus(sib || parent, { scroll: false });
-                        tree.delete(node);
-                    }
-                }
                 return;
             }
             if (e.key === "Tab" && !e.shiftKey) {
diff --git a/dist/module/interfaces/node-api.js b/dist/module/interfaces/node-api.js
index 484fe0a..1321d59 100644
--- a/dist/module/interfaces/node-api.js
+++ b/dist/module/interfaces/node-api.js
@@ -2,7 +2,7 @@ import { ROOT_ID } from "../data/create-root";
 export class NodeApi {
     constructor(params) {
         this.handleClick = (e) => {
-            if (e.metaKey && !this.tree.props.disableMultiSelection) {
+            if ((e.metaKey || e.ctrlKey) && !this.tree.props.disableMultiSelection) {
                 this.isSelected ? this.deselect() : this.selectMulti();
             }
             else if (e.shiftKey && !this.tree.props.disableMultiSelection) {
