diff --git a/dist/main/components/default-row.js b/dist/main/components/default-row.js
index a9e392128a0c63207fa2b3180a870915fd5f2a28..8cf0e2cfd1e65bcdd06d515f5ea5ad23a9744704 100644
--- a/dist/main/components/default-row.js
+++ b/dist/main/components/default-row.js
@@ -3,6 +3,11 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.DefaultRow = void 0;
 const jsx_runtime_1 = require("react/jsx-runtime");
 function DefaultRow({ node, attrs, innerRef, children, }) {
-    return ((0, jsx_runtime_1.jsx)("div", Object.assign({}, attrs, { ref: innerRef, onFocus: (e) => e.stopPropagation(), onClick: node.handleClick, children: children })));
+    const handleRowClick = (e) => {
+        if (e.defaultPrevented)
+            return;
+        node.handleClick(e);
+    };
+    return ((0, jsx_runtime_1.jsx)("div", Object.assign({}, attrs, { ref: innerRef, onFocus: (e) => e.stopPropagation(), onClick: handleRowClick, children: children })));
 }
 exports.DefaultRow = DefaultRow;
diff --git a/dist/main/interfaces/node-api.js b/dist/main/interfaces/node-api.js
index 120c7388c24b757705aba64b4d16f36cde93accf..0eb035a6787ceee090f01135f153a0825ea6f89b 100644
--- a/dist/main/interfaces/node-api.js
+++ b/dist/main/interfaces/node-api.js
@@ -5,7 +5,7 @@ const create_root_1 = require("../data/create-root");
 class NodeApi {
     constructor(params) {
         this.handleClick = (e) => {
-            if (e.metaKey && !this.tree.props.disableMultiSelection) {
+            if ((e.metaKey || e.ctrlKey) && !this.tree.props.disableMultiSelection) {
                 this.isSelected ? this.deselect() : this.selectMulti();
             }
             else if (e.shiftKey && !this.tree.props.disableMultiSelection) {
diff --git a/dist/module/components/default-container.js b/dist/module/components/default-container.js
index 47724f59b482454fe3144dbb98bd16d3df6a9c17..2285e35ea0073a773b7b74e22758056fd3514c1a 100644
--- a/dist/module/components/default-container.js
+++ b/dist/module/components/default-container.js
@@ -34,28 +34,6 @@ export function DefaultContainer() {
                 return;
             }
             if (e.key === "Backspace") {
-                if (!tree.props.onDelete)
-                    return;
-                const ids = Array.from(tree.selectedIds);
-                if (ids.length > 1) {
-                    let nextFocus = tree.mostRecentNode;
-                    while (nextFocus && nextFocus.isSelected) {
-                        nextFocus = nextFocus.nextSibling;
-                    }
-                    if (!nextFocus)
-                        nextFocus = tree.lastNode;
-                    tree.focus(nextFocus, { scroll: false });
-                    tree.delete(Array.from(ids));
-                }
-                else {
-                    const node = tree.focusedNode;
-                    if (node) {
-                        const sib = node.nextSibling;
-                        const parent = node.parent;
-                        tree.focus(sib || parent, { scroll: false });
-                        tree.delete(node);
-                    }
-                }
                 return;
             }
             if (e.key === "Tab" && !e.shiftKey) {
diff --git a/dist/module/components/default-row.js b/dist/module/components/default-row.js
index 3540f228690b6bf7499cebe48b233c9920dfd807..9bae5a80531121c905d390e9c14dbf009d8cabbc 100644
--- a/dist/module/components/default-row.js
+++ b/dist/module/components/default-row.js
@@ -1,4 +1,9 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 export function DefaultRow({ node, attrs, innerRef, children, }) {
-    return (_jsx("div", Object.assign({}, attrs, { ref: innerRef, onFocus: (e) => e.stopPropagation(), onClick: node.handleClick, children: children })));
+    const handleRowClick = (e) => {
+        if (e.defaultPrevented)
+            return;
+        node.handleClick(e);
+    };
+    return (_jsx("div", Object.assign({}, attrs, { ref: innerRef, onFocus: (e) => e.stopPropagation(), onClick: handleRowClick, children: children })));
 }
diff --git a/dist/module/interfaces/node-api.js b/dist/module/interfaces/node-api.js
index 484fe0a6fb5f0821741a50b8a06720f01e67c478..1321d59e136d63e62c5015ccfe130c59881f2079 100644
--- a/dist/module/interfaces/node-api.js
+++ b/dist/module/interfaces/node-api.js
@@ -2,7 +2,7 @@ import { ROOT_ID } from "../data/create-root";
 export class NodeApi {
     constructor(params) {
         this.handleClick = (e) => {
-            if (e.metaKey && !this.tree.props.disableMultiSelection) {
+            if ((e.metaKey || e.ctrlKey) && !this.tree.props.disableMultiSelection) {
                 this.isSelected ? this.deselect() : this.selectMulti();
             }
             else if (e.shiftKey && !this.tree.props.disableMultiSelection) {
